Doctrine/UserManager.php

```diff
@@ -28,30 +28,41 @@ class UserManager extends BaseUserManager
     /**
      * @var string
      */
+    private $className;
+
+    /**
+     * @var string
+     */
     protected $class;
```

Comment:
the property storing the actual class name must become private (forcing child classes to access it through ``getClass()``, as it is not resolved in the constructor anymore